### ISSUE TITLE
cannot run training on FFHQ with batchsize 64 and image size 48

### ISSUE THREAD

**Issue by LigZhong** [raywzy/ICT#13]
I have 8 V100 gpus and run the training code without error with image size 32 and batch size 64 but cuda out of memory occurs when I set image size to 48. Anyone met this?

**Comment by LigZhong** [comment]
> Hi, Are your V100 GPUs 32G?

Yes, I used V100 GPUs 32G. I checked that no other process occupied the GPUs.

**Comment by raywzy** [comment]
Have you tried other batch size setting, like 48?
